src/uct/ib/base/ib_iface.c

```diff
@@ -865,6 +865,8 @@ UCS_CLASS_INIT_FUNC(uct_ib_iface_t, uct_ib_iface_ops_t *ops, uct_md_h md,
         goto err;
     }
 
+    self->is_roce_v2 = is_roce_v2;
```

Comment:
can we remove the local var is_roce_v2 and use self->is_roce_v2 in all places instead?